core/autoload/spacevim.vim

```diff
@@ -145,11 +145,18 @@ function! spacevim#end() abort
   call s:config()
   if exists('*UserConfig') | call UserConfig() | endif
 
-  try
-    execute 'helptags' g:spacevim.base . '/core/doc'
-  catch
-    echom v:exception
-  endtry
+  let helptag_file = g:spacevim.base . '/core/doc/spacevim.txt'
+  let helptag_time = getftime(helptag_file)
+  let helptag_lastprocessed_file = g:spacevim.base . '/core/doc/.spacevim_lastprocessed'
```

Comment:
I'm thinking of using whatever like `.spacevim_last_helptags_run` might be easier to understand what it is.

Reply:
Good idea, renamed it.